src/dmd/func.d

```diff
@@ -2410,19 +2410,19 @@ void MODMatchToBuffer(OutBuffer* buf, ubyte lhsMod, ubyte rhsMod)
     bool sharedMismatchOnly = ((lhsMod ^ rhsMod) == MODFlags.shared_);
 
     if (lhsMod & MODFlags.shared_)
-        buf.writestring("shared ");
+        buf.writestring("`shared` ");
     else if (sharedMismatch && !(lhsMod & MODFlags.immutable_))
-        buf.writestring("non-shared ");
+        buf.writestring("non-`shared` ");
```

Comment:
please no highlighting parts of words

Reply:
Would you prefer \`non-shared`, or just not highlighted at all?

Reply:
I went with `non-shared`, `impure`, `non-@nogc`, etc..

Reply:
Just non-shared, impure, etc., please